File: backend/GeoQuiz_backend/GeoQuiz_backend/DTOs/PvP/SubmitAnswerRequest.cs
namespace GeoQuiz_backend.DTOs.PvP
{
    public class SubmitAnswerRequest
    {
        public Guid MatchId { get; set; }           
        public string QuestionId { get; set; } = null!;
        public int SelectedIndex { get; set; }
        public int TimeSpentMs { get; set; }
        public int QuestionNumber { get; set; }
    }
}

File: frontend/app/src/main/java/com/example/geoquiz_frontend/data/remote/dtos/SubmitAnswerRequest.java
package com.example.geoquiz_frontend.data.remote.dtos;

public class SubmitAnswerRequest {
    private String matchId;
    private String questionId;
    private int selectedIndex;
    private int timeSpentMs;
    private int questionNumber;

    public SubmitAnswerRequest(String matchId, String questionId,
                               int selectedIndex, int timeSpentMs, int questionNumber) {
        this.matchId = matchId;
        this.questionId = questionId;
        this.selectedIndex = selectedIndex;
        this.timeSpentMs = timeSpentMs;
        this.questionNumber = questionNumber;
    }

    public String getMatchId() { return matchId; }
    public String getQuestionId() { return questionId; }
    public int getSelectedIndex() { return selectedIndex; }
    public int getTimeSpentMs() { return timeSpentMs; }
    public int getQuestionNumber() { return questionNumber; }
}